Product flavor with duplicate exportedDeps

I am trying okbuck for the first time in a pretty big app.
I am using `com.uber:okbuck:0.14.3`

When I run `./buckw targets` I get:

```
BUILD SUCCESSFUL

Total time: 9.274 secs
Task timings:
   1593ms  :okbuckClean
    100ms  :setupOkbuck
   2328ms  :MyApp:okbuck

 PROCEEDING WITH BUCK 

Not using buckd because watchman isn't installed.
[+] PARSING BUCK FILES...25.7s
BUILD FAILED: //MyApp:src_productFlavorBuildType: parameter 'exportedDeps': duplicate element "//MyApp:productFlavorBuildType_aidls"
```

Inside the BUCK generated file I see:

```
android_library(
	name = 'src_productFlavorBuildType',
	srcs = glob([
		'src-gen-dao/**/*.java',
		'src/**/*.java',
		'build-types/buildtype/java/**/*.java',
	]),
	manifest = 'build/okbuck/productFlavorBuildType/AndroidManifest.xml',
	annotation_processors = [
		'butterknife.compiler.ButterKnifeProcessor',
		'com.bluelinelabs.logansquare.processor.JsonAnnotationProcessor',
	],
	annotation_processor_deps = [
		':apt_jar_productFlavorBuildType',
	],
	exported_deps = [
		':productFlavorBuildType_aidls',
		':productFlavorBuildType_aidls',
	],
	source = '7',
	target = '7',

...
```

Any idea what could be the problem?

--- a/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy
+++ b/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy
@@ -26,8 +26,8 @@ abstract class AndroidBuckRuleComposer extends JavaBuckRuleComposer {
         return "prebuilt_native_library_${target.name}_${jniLibDir.replaceAll("/", "_")}"
     }
 
-    static String aidl(AndroidTarget target) {
-        return "${target.name}_aidls"
+    static String aidl(AndroidTarget target, String aidlDir) {
+        return "${target.name}_${aidlDir}_aidls"
     }
 
     static String appLib(AndroidTarget target) {

--- a/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy
+++ b/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy
@@ -27,7 +27,7 @@ abstract class AndroidBuckRuleComposer extends JavaBuckRuleComposer {
     }
 
     static String aidl(AndroidTarget target, String aidlDir) {
-        return "${target.name}_${aidlDir.replaceAll('-', '_')}_aidls"
+        return "${target.name}_${aidlDir.replaceAll("[/-]", '_')}_aidls"
     }
 
     static String appLib(AndroidTarget target) {

--- a/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy
+++ b/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidBuckRuleComposer.groovy
@@ -27,7 +27,7 @@ abstract class AndroidBuckRuleComposer extends JavaBuckRuleComposer {
     }
 
     static String aidl(AndroidTarget target, String aidlDir) {
-        return "${target.name}_${aidlDir}_aidls"
+        return "${target.name}_${aidlDir.replaceAll('-', '_')}_aidls"
     }
 
     static String appLib(AndroidTarget target) {

--- a/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/GenAidlRuleComposer.groovy
+++ b/buildSrc/src/main/groovy/com/uber/okbuck/composer/android/GenAidlRuleComposer.groovy
@@ -10,7 +10,7 @@ final class GenAidlRuleComposer extends AndroidBuckRuleComposer {
     }
 
     static GenAidlRule compose(AndroidTarget target, String aidlDir) {
-        return new GenAidlRule(aidl(target), aidlDir, "${target.path}/${aidlDir}", target.manifest,
+        return new GenAidlRule(aidl(target, aidlDir), aidlDir, "${target.path}/${aidlDir}", target.manifest,
                 targets(target.main.targetDeps))
     }
 }

